### DISCUSSION TITLE
有人打算增加更多的leetcode us 的支持吗

### DISCUSSION THREAD

**Discussion by wnykuang** [ccagml/leetcode-extension#293] [General]
好像没有美版的每日一题 英文支持也不太好
要是没人做我就来做了

**Comment by ccagml** [comment]
感觉好像没什么人在这插件用us

**Reply by wnykuang** [reply]
你这比leetcode 官方那个vscode 插件的更新维护好太多了
我就拿来用来在us 上实操
那玩意的tag 都是18年的东西了

**Comment by wnykuang** [comment]
leetcode - us 的搜索也有问题。。。
